.circleci/config.yml

```diff
@@ -44,10 +41,7 @@ jobs:
       - restore_cache: *restore-yarn-cache
       - restore_cache: *restore-node-modules-cache
       - run: sudo corepack enable
-      - run: yarn install --non-interactive --cache-folder ~/.cache/yarn
-      - run:
-          name: Check if yarn.lock changed during install
-          command: git diff --exit-code
+      - run: yarn install --immutable
```

Comment:
note that `--immutable` does the git diff check for us and fails if the lockfile changes during install 👍🏽